xds/build.gradle

```diff
@@ -47,7 +47,11 @@ dependencies {
             libraries.guava_testlib,
             libraries.netty_epoll
 
+    // Use grpc-netty-shaded for transitive dependency.
+    compileOnly project(':grpc-netty')
     shadow project(path: ':grpc-netty-shaded', configuration: 'shadow')
+    shadow configurations.compile.getDependencies()
```

Comment:
This brings in library.pgv. It needs to be excluded. (maybe with minus() like used in grpc-all?)

I sort of think excluding may be better than using compileOnly for grpc-netty, since then both 'compile' configuration and 'shadow' configuration are functional outside of this project. It also seems to be a bit less likely to have surprises. But I can accept what's been done here.

Reply:
Done.